### ISSUE TITLE
HoeffdingTree can return probabilities are all zero.

### ISSUE THREAD

**Issue by PGijsbers** [scikit-multiflow/scikit-multiflow#64]
### Expected behaviour

It is possible that no classes get a vote fr
https://github.com/scikit-multiflow/scikit-multiflow/blob/master/src/skmultiflow/trees/hoeffding_tree.py#L1097-L1105

**Comment by PGijsbers** [comment]
This is of course assuming we always want class probabilities to sum to 1, which in my opinion they should. If there is a good reason not to do this, I would be interested to hear why :)

Again, I'll happily set up a PR with the fix. I just figured it would be best to know what the desired fix is (else-clause or `modifying normalize_values_in_dict`).

**Comment by jacobmontiel** [comment]
There is the case where a tree is empty. In such case we return 0.0 basically because there is no enough information.

**Comment by jacobmontiel** [comment]
Yes, different to batch learning it is unlikely that the classes distribution is known in advance in Streams. Additionally, to use the priors we would need to extend the Stream class to include such information and handle it properly, in other words, this is a framework-level consideration.
